tools/accuracy_checker/accuracy_checker/data_readers/data_reader.py

```diff
@@ -351,6 +351,13 @@ class NumPyReader(BaseReader):
     def read(self, data_id):
         return np.load(str(self.data_source / data_id))
 
+class OAR3DReader(BaseReader):
+    __provider__ = 'oar3d_reader'
+
+    def read(self, data_id):
+        data = np.load(str(self.data_source / data_id))
+        return data['inputs']
```

Comment:
please extend numpy reader to work with dict files (optional key parameter)

Reply:
Numpy reader extended, variable naming control has been moved to configuration file